README.md

```diff
@@ -10,11 +10,12 @@
 [![License][license-badge]][license-url]
 [![Sourcegraph][sg-badge]][sg-url]
 
-| Branch  | Tests                                    | Linting                               |
-|---------|------------------------------------------|---------------------------------------|
-| main    | [![Tests][tests-badge]][tests-url]       | [![Lint][lint-badge]][lint-url]       |
-| v0.37.x | [![Tests][tests-badge-v037x]][tests-url] | [![Lint][lint-badge-v037x]][lint-url] |
-| v0.34.x | [![Tests][tests-badge-v034x]][tests-url] | [![Lint][lint-badge-v034x]][lint-url] |
+| Branch  | Tests                                          | Linting                                     |
+|---------|------------------------------------------------|---------------------------------------------|
+| main    | [![Tests][tests-badge]][tests-url]             | [![Lint][lint-badge]][lint-url]             |
+| v0.38.x | [![Tests][tests-badge-v038x]][tests-url-v038x] | [![Lint][lint-badge-v038x]][lint-url-v038x] |
```

Comment:
Wanted to highlight these images don't render, probably because the tests for v0.38 are not active yet?

Reply:
As expected, now that #596 has been merged, the images render.